### ISSUE TITLE
Jedis.hset(String, Map<String, Object> is not available in Jedis API

### ISSUE THREAD

**Issue by shannonantony** [RediSearch/JRediSearch#116]
Hi,

I am not able to run the sample as per https://github.com/RediSearch/JRediSearch example with respect to 

try(Jedis conn = client.connction()){
	conn.hset("item", fields);
}

As per Jedis API, fields should be a Map<String, String>

can you please clarify and help

thanks

Jay

**Comment by gkorland** [comment]
Yes field should be a `Map<String, String>`.
e.g.
```java
Map<String, String> map = new HashMap<>();
map.put("first", "Ron")
map.put("last", "Kor")
```

**Comment by shannonantony** [comment]
Thanks @gkorland . Does that mean we can not use this API to write/read Numeric fields which are defined in Redisearch Schema?
How do we handle this scenario of dealing with non text fields which are part of schema?

thanks

Jay

**Comment by gkorland** [comment]
@shannonantony you're right I guess the Jedis API could be better, but in your case all you need to do is to 

```
Map<String, String> fields = new HashMap<>();
map.put("first", "Ron")
map.put("last", "Kor")
map.put("age", String.valueOf​(age))
```

**Comment by shannonantony** [comment]
Thank you very much for quick response @gkorland . That is what I was thinking too as a work around, but I just wanted to double check. I will test it out and see.
